discovery-infra/tests/base_test.py

```diff
@@ -228,3 +239,12 @@ def update_oc_config(self, nodes, cluster):
         infra_utils.config_etc_hosts(cluster_name=cluster.name,
                                base_dns_domain=env_variables["base_domain"],
                                api_vip=api_vip)
+
+    @staticmethod
+    def get_insufficeint_disk_size_for_nodes(api_client):
+        min_host_requirements = api_client.get_host_requirements()
+        min_disk = min_host_requirements.worker.disk_size_gb
+        min_disk_bytes = min_disk * pow(1000, 3)
+        insufficient_disk_bytes = math.floor(min_disk_bytes * 0.90)
+
+        return {'worker_disk': insufficient_disk_bytes, 'master_disk': insufficient_disk_bytes}
```

Comment:
why we need to return same value twice? cant we just return node_disk?

Reply:
`worker_disk` and `master_disk` are the names of the terrafrom variables. This makes it easier to pass to override_vm_parameters

Reply:
think its better to have this method in test level. WDYT?

Reply:
done